ansible/roles/grafana/tasks/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: mkdir for config and data
+  file:
+    path: "{{ item }}"
+    state: directory
+    mode: "u=rwx,g=rx,o="
+  with_items:
+    - "{{ grafana_config_path }}"
+    - "{{ grafana_lib_path }}"
```

Comment:
Grafana currently runs inside of a docker container. I am not sure what is the best way to setup permissions properly for this. In theory since we map volumes the grafana process should actually not have access to anything else.